src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size } from './theme';
+import { P } from './text';
+import CardList from './card-list';
+import useAllVideos from '../../hooks/use-all-videos';
+import usePodcasts from '../../hooks/use-podcasts';
+
+const Tab = styled('div')`
+    overflow: hidden;
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    margin-left: 38px;
+    margin-right: 38px;
+    margin-top: 50px;
+    margin-bottom: 30px;
+`;
+
+const TabButton = styled('button')`
+    float: right;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    transition: 0.3s;
+    font-size: 18px;
+    font-family: Fira Mono;
+    color: ${colorMap.greyDarkOne};
+    background-color: inherit;
+    border-bottom: 3px solid transparent;
+    width: 136px;
+    padding-top: ${size.micro};
+    padding-bottom: ${size.medium};
+    padding-left: ${size.mediumLarge};
+    padding-right: ${size.mediumLarge};
+    &:hover {
+        color: white;
+    }
+    &:focus {
+        border-bottom-color: ${colorMap.darkGreen};
+    }
+`;
+
+const AllContentStyle = {
+    marginRight: '818px',
```

Comment:
This is where the responsive issues are coming from! What happens is sometimes we run into a situation where this space isn't available and we get wonky behavior, but the browser is correct in thinking it needs to put this `818px` away.

We'll talk about how we can remedy this in the check in, but let's use something like [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or css-grid to provide a more responsive layout!